core/src/test/java/org/elasticsearch/index/query/MatchQueryBuilderTests.java

```diff
@@ -277,71 +277,6 @@ public void testBadAnalyzer() throws IOException {
         }
     }
 
-    public void testPhrasePrefixMatchQuery() throws IOException {
-        String json1 = "{\n" +
-                "    \"match_phrase_prefix\" : {\n" +
-                "        \"message\" : \"this is a test\"\n" +
-                "    }\n" +
-                "}";
-
-        String expected = "{\n" +
-                "  \"match\" : {\n" +
-                "    \"message\" : {\n" +
-                "      \"query\" : \"this is a test\",\n" +
-                "      \"type\" : \"phrase_prefix\",\n" +
-                "      \"operator\" : \"OR\",\n" +
-                "      \"slop\" : 0,\n" +
-                "      \"prefix_length\" : 0,\n" +
-                "      \"max_expansions\" : 50,\n" +
-                "      \"fuzzy_transpositions\" : true,\n" +
-                "      \"lenient\" : false,\n" +
-                "      \"zero_terms_query\" : \"NONE\",\n" +
-                "      \"boost\" : 1.0\n" +
-                "    }\n" +
-                "  }\n" +
-                "}";
-        MatchQueryBuilder qb = (MatchQueryBuilder) parseQuery(json1);
-        checkGeneratedJson(expected, qb);
-
-        String json2 = "{\n" +
-                "    \"match\" : {\n" +
-                "        \"message\" : {\n" +
-                "            \"query\" : \"this is a test\",\n" +
-                "            \"type\" : \"phrase_prefix\"\n" +
-                "        }\n" +
-                "    }\n" +
-                "}";
```

Comment:
From what I can see the was only testing `match_phrase_prefix` queries but with both of the old forms (specifying `match_phrase_prefix` as the query name and specifying `match` as the query name with `"type": "phrase_prefix"`. Only the former is supported in the new query type which is why the latter is not there